ci/deploy.sh

```diff
@@ -58,6 +58,7 @@ parse_py_version() {
   version="${version//-alpha/a}"
   version="${version//-beta/b}"
   version="${version//-rc/rc}"
+  version="${version//-//}"
```

Comment:
I think -rc becomes redundant.

Reply:
It's a convention to name release candicate as `v$major}.${minor}.${fix}-rc{number}`

Reply:
Sometimes we have tag like `v0.12.1-a1`. Such patterns are not handled by above lines